scripts/workers/rl_submission_worker.py

```diff
@@ -164,9 +164,26 @@ def get_api_object(cluster_name, cluster_endpoint, challenge, evalai):
     return api_instance
 
 
-def get_core_v1_api_object(cluster_name, challenge, evalai):
+def get_instance(cluster_name, cluster_endpoint, challenge, evalai):
```

Comment:
Can we please have a meaningful name for this function?

Reply:
@RishabhJain2018  Done